OVERVIEW.rst

```diff
@@ -0,0 +1,17 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+.. _concepts:
+
+Overview
+========
+The purpose of this package is to establish an entrypoint in your application,
+providing through this the RESTful serving of cropped images according to the
+IIIF library. A previewing functionality is also provided for images and pdf
+files.
+
+In the **Usage** section there are examples provided for the above cases.
```

Comment:
I would follow the standard Invenio documentation approach. Overview could be added to ``README.rst`` and included(``.. include:: ../README.rst``) in ``index.rst`` as in other Invenio packages. 

A few examples:
- https://github.com/inveniosoftware/invenio-search/blob/master/README.rst
- https://github.com/inveniosoftware/invenio-records-files/blob/master/README.rst

What do you think?